.github/workflows/examples.yml

```diff
@@ -0,0 +1,128 @@
+        cfg:
+          - python-version: 3.7
+            rdkit: "true"
+            openeye: "false"
+
+          - python-version: 3.7
+            rdkit: "false"
+            openeye: "true"
+
+
+    env:
+      CI_OS: ${{ matrix.os }}
+      RDKIT: ${{ matrix.cfg.rdkit }}
+      OPENEYE: ${{ matrix.cfg.openeye }}
+      PYVER: ${{ matrix.cfg.python-version }}
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+      PACKAGE: openff
+      PYTEST_ARGS: -r fE --tb=short --cov=openff --cov-config=setup.cfg --cov-append --cov-report=xml
+      NB_ARGS: -v --nbval-lax --ignore=examples/deprecated
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        name: Install only RDKit
+        if: ${{ matrix.cfg.rdkit == 'TRUE' && matrix.cfg.openeye == 'FALSE' }}
+        with:
+          python-version: ${{ matrix.cfg.python-version }}
+          activate-environment: test
+          environment-file: devtools/conda-envs/rdkit.yaml
+          auto-activate-base: false
+      - uses: conda-incubator/setup-miniconda@v2
+        name: Install with OpenEye toolkits
+        if: ${{ matrix.cfg.rdkit == 'FALSE' && matrix.cfg.openeye == 'TRUE' }}
+        with:
+          python-version: ${{ matrix.cfg.python-version }}
+          activate-environment: test
+          environment-file: devtools/conda-envs/openeye.yaml
+          auto-activate-base: false
+
+      - name: Install additional example dependencies
+        shell: bash -l {0}
+        run: |
+          conda env update --file examples/environment.yaml --name test
+
+      - name: Additional info about the build
+        shell: bash -l {0}
+        run: |
+          uname -a
+          df -h
+          ulimit -a
+
+      - name: Make oe_license.txt file from GH org secret "OE_LICENSE"
+        shell: bash
+        env:
+          OE_LICENSE_TEXT: ${{ secrets.OE_LICENSE }}
+        run: |
+          echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
+
+      - name: Install package
+        shell: bash -l {0}
+        run: |
+          python setup.py develop --no-deps
+
+      - name: Environment Information
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
+
+      - name: Run example scripts without OE
+        shell: bash -l {0}
+        run: |
+          if [[ "$RDKIT" == false ]]; then
+            PYTEST_ARGS+=" --ignore=examples/check_dataset_parameter_coverage"
+            PYTEST_ARGS+=" --ignore=examples/QCArchive_interface"
+          fi
+          pytest $PYTEST_ARGS openff/toolkit/tests/test_examples.py
+
+      - name: Run example notebooks
+        shell: bash -l {0}
+        run: |
+          pytest $NB_ARGS examples
+
+      - name: Conda tree information
+        shell: bash -l {0}
+        run: |
+          conda install -c conda-forge conda-tree
+          conda tree deptree
+          conda tree leaves
+          conda tree whoneeds rdkit
```

Comment:
This is slick. I'm going to use `conda tree` for debugging from now on!

Reply:
I only wish it could run the solver and give the tree for a hypothetical environment...

Reply:
Also it sucks that it seem to be broken on Linux...